### DISCUSSION TITLE
⭐️ Starstruck Badge Exchange ⭐️

### DISCUSSION THREAD

**Discussion by the1Riddle** [drknzz/GitHub-Achievements#274] [Q&A]
hello folks,
Please star my repo that contains the project i've done, i will star yours in exchange when you reply with the link to the repo you want me to star.

here is the link:👉  [my repo](https://github.com/rezzcode/zoea-opensource)

**thanks** ❤️

**Comment by ternera** [comment]
Please star my repository and I will return it: https://github.com/ternera/hacker101-ctf

**Reply by the1Riddle** [reply]
done

**Reply by RomainMarques** [reply]
Please star my repo : https://github.com/RomainMarques/SoftwareForCloud

**Reply by the1Riddle** [reply]
> Please star my repo : https://github.com/RomainMarques/SoftwareForCloud

@RomainMarques thanks, i have stared yours too. A good project you have. :smiley_cat:

**Reply by 1wos** [reply]
@RomainMarques I stared your repo please return my repo !! 👍 https://github.com/sommizzu/ASF-DataAnalysis

**Reply by RomainMarques** [reply]
@sommizzu I stared it, thank you!

**Reply by RomainMarques** [reply]
@Maltered I stared your repo, don't forget to stare mine please!

**Reply by ganochenkodg** [reply]
@RomainMarques done, star mine pls https://github.com/ganochenkodg/qdrant-operator

**Reply by RomainMarques** [reply]
Done 👍

**Reply by KamBRJC** [reply]
@sommizzu  star back https://github.com/z-nnovation/GitSwitcher

**Reply by iv1teq** [reply]
fr bro

**Reply by MatteoMgr2008** [reply]
@ternera
@the1Riddle
star mine i stared yours:
https://github.com/MatteoMgr2008/Arduino-ROBOTICA-E-AI

**Reply by the1Riddle** [reply]
> @MatteoMgr2008 star mine i stared yours: https://github.com/MatteoMgr2008/Arduino-ROBOTICA-E-AI

I got no readme!!😄, done though. much appreciated. 💪

**Reply by MatteoMgr2008** [reply]
> > @MatteoMgr2008 star mine i stared yours: https://github.com/MatteoMgr2008/Arduino-ROBOTICA-E-AI
> 
> I got no readme!!😄, done though. much appreciated. 💪

I have never posted a readme because they are just some example exercises in Italian on Arduino and it is not a big computer programming project. Anyway, thank you so much for starring my repo!

**Reply by Istituto-freudinttheprodev** [reply]
@the1Riddle
@ternera
@KamBRJC
@iv1teq
star mine please I starred yours:
https://github.com/Istituto-freudinttheprodev/Sito-web-lavoro-di-gruppo-INFORMATICA

**Reply by spidybroz** [reply]
## ⭐ Star & 🍴 Fork the Repo <3

If you like this project, please **star** ⭐ the repo — I’ll star yours back!  
Feel free to **fork** 🍴 it, contribute, or share it with others 🚀

### 🔗 Links
- **GitHub:** https://github.com/maiz-an/DevTunnel-CLI  
- **Website:** https://devtunnel-cli.vercel.app

**Comment by the1Riddle** [comment]
@ganochenkodg , thank you for starring my repo. I have starred yours as well.

**Reply by the1Riddle** [reply]
just to add, a good project you have there 😄

**Reply by the1Riddle** [reply]
> @Istituto-freudinttheprodev star mine please I starred yours: https://github.com/Istituto-freudinttheprodev/Sito-web-lavoro-di-gruppo-INFORMATICA

Nice work, buddy. done.

**Reply by Istituto-freudinttheprodev** [reply]
> > @Istituto-freudinttheprodev star mine please I starred yours: https://github.com/Istituto-freudinttheprodev/Sito-web-lavoro-di-gruppo-INFORMATICA
> 
> Nice work, buddy. done.

Thanks a lot bro, have a good day!

**Comment by peterbaccaro** [comment]
Please give me a star for my repo and help me get ⭐️ starstruck ⭐️!

https://github.com/peterbaccaro/design-patterns-python

Much appreciated. Thanks in advance.

**Reply by the1Riddle** [reply]
done, consider checking mine too

**Reply by ganochenkodg** [reply]
@peterbaccaro done, and mine pls https://github.com/ganochenkodg/qdrant-operator

**Reply by spidybroz** [reply]
## ⭐ Star & 🍴 Fork the Repo <3

If you like this project, please **star** ⭐ the repo — I’ll star yours back!  
Feel free to **fork** 🍴 it, contribute, or share it with others 🚀

### 🔗 Links
- **GitHub:** https://github.com/maiz-an/DevTunnel-CLI  
- **Website:** https://devtunnel-cli.vercel.app

**Comment by Coolbreeze151** [comment]
Hi there,

Appreciate the exchange of stars here too. Thank you!

https://github.com/Coolbreeze151/BackendRoadmap

**Reply by ganochenkodg** [reply]
@Coolbreeze151 @KamBRJC done, and mine pls https://github.com/ganochenkodg/qdrant-operator

**Reply by the1Riddle** [reply]
done

**Reply by spidybroz** [reply]
## ⭐ Star & 🍴 Fork the Repo <3

If you like this project, please **star** ⭐ the repo — I’ll star yours back!  
Feel free to **fork** 🍴 it, contribute, or share it with others 🚀

### 🔗 Links
- **GitHub:** https://github.com/maiz-an/DevTunnel-CLI  
- **Website:** https://devtunnel-cli.vercel.app

**Comment by maiz-an** [comment]
### if you visit my site will be index on google so, please
https://www.mzieos.com/

**Reply by the1Riddle** [reply]
> ### if you visit my site will be index on google so, please
> https://www.mzieos.com/

I'd just add it on a [Wishlist](https://github.com/mzieos)

**Reply by MatteoMgr2008** [reply]
> ### if you visit my site will be index on google so, please
> https://www.mzieos.com/

@maiz-an I also visited your website, I hope this helps! Just out of curiosity, what will the website be about when it's finished?

**Reply by maiz-an** [reply]
> > ### if you visit my site will be index on google so, please
> 
> > https://www.mzieos.com/
> 
> 
> 
> @maiz-an I also visited your website, I hope this helps! Just out of curiosity, what will the website be about when it's finished?

ASAP @MatteoMgr2008 
You can see what under development 
From this 
https://dev-maizan.vercel.app

**Reply by maiz-an** [reply]
> > ### if you visit my site will be index on google so, please
> 
> > https://www.mzieos.com/
> 
> 
> 
> I'd just add it on a [Wishlist](https://github.com/mzieos)

Thanks

**Reply by the1Riddle** [reply]
> > @maiz-an I also visited your website, I hope this helps! Just out of curiosity, what will the website be about when it's finished?
> 
> ASAP @MatteoMgr2008 You can see what under development From this https://dev-maizan.vercel.app

nice for a start 💪

**Reply by maiz-an** [reply]
> > > @maiz-an I also visited your website, I hope this helps! Just out of curiosity, what will the website be about when it's finished?
> 
> > 
> 
> > ASAP @MatteoMgr2008 You can see what under development From this https://dev-maizan.vercel.app
> 
> 
> 
> nice for a start 💪 
> 
> 

@the1Riddle thank you

**Reply by MatteoMgr2008** [reply]
> > > ### if you visit my site will be index on google so, please
> > 
> > 
> > > https://www.mzieos.com/
> > 
> > 
> > @maiz-an I also visited your website, I hope this helps! Just out of curiosity, what will the website be about when it's finished?
> 
> ASAP @MatteoMgr2008 You can see what under development From this https://dev-maizan.vercel.app

Thanks for replying! I saw your website from your link, and I have to say it's very nice looking and, above all, very interesting. Did you program it with the Vercel framework, right? How's it? Is it difficult to use? In the past, I've only used Streamlit in Python as a framework!

**Reply by MatteoMgr2008** [reply]
> > > @maiz-an I also visited your website, I hope this helps! Just out of curiosity, what will the website be about when it's finished?
> > 
> > 
> > ASAP @MatteoMgr2008 You can see what under development From this https://dev-maizan.vercel.app
> 
> nice for a start 💪

It's true, you're absolutely right, his star on the repo is more deserved

**Reply by maiz-an** [reply]
@MatteoMgr2008 thanks for your words. It's really a lot,
It's a react+vite project i built from scratch and i hosted via vercelapp.

**Reply by MatteoMgr2008** [reply]
@maiz-an Wow, what a difficult and complex project! You really are good at this. I've never done any of this myself, but if I ever have to do it in the future for some reason, I'll definitely take your repo into consideration to learn. You've truly created an extraordinary project; you clearly have experience in the field!

**Reply by maiz-an** [reply]
> @maiz-an Wow, what a difficult and complex project! You really are good at this. I've never done any of this myself, but if I ever have to do it in the future for some reason, I'll definitely take your repo into consideration to learn. You've truly created an extraordinary project; you clearly have experience in the field!

@MatteoMgr2008 thanks a lot for your words

**Reply by MatteoMgr2008** [reply]
Keep it up, I recommend you, have a good day!

**Reply by maiz-an** [reply]
> Keep it up, I recommend you, have a good day!

You made my day🙌🏻

**Reply by MatteoMgr2008** [reply]
> > Keep it up, I recommend you, have a good day!
> 
> You made my day🙌🏻

Thanks for the compliment, I'm happy now too!

**Reply by spidybroz** [reply]
> > > Keep it up, I recommend you, have a good day!
> > 
> > 
> > You made my day🙌🏻
> 
> Thanks for the compliment, I'm happy now too!

hi bro its me @maizian


## ⭐ Star & 🍴 Fork the Repo <3

If you like this project, please **star** ⭐ the repo — I’ll star yours back!  
Feel free to **fork** 🍴 it, contribute, or share it with others 🚀

### 🔗 Links
- **GitHub:** https://github.com/maiz-an/DevTunnel-CLI  
- **Website:** https://devtunnel-cli.vercel.app

**Comment by MatteoMgr2008** [comment]
@the1Riddle
star mine i stared yours:
https://github.com/MatteoMgr2008/Arduino-ROBOTICA-E-AI

**Reply by the1Riddle** [reply]
done

**Reply by MatteoMgr2008** [reply]
Thanks so much for starring my repo, I did the same with yours!

**Reply by Istituto-freudinttheprodev** [reply]
@the1Riddle
star mine please I starred yours:
https://github.com/Istituto-freudinttheprodev/Sito-web-lavoro-di-gruppo-INFORMATICA

**Reply by malmorox** [reply]
done! star my repo pls https://github.com/malmorox/Portfolio

**Comment by Axle-Bucamp** [comment]
hey nice to meet you all haha :
https://github.com/Axle-Bucamp/projectHosting
star my repo and i star yours

**Reply by MatheusFarias8** [reply]
done.

https://github.com/MatheusFarias8/Multiple-Clients-Chat-Java

**Reply by malmorox** [reply]
done! star my repo pls https://github.com/malmorox/Portfolio

**Reply by malmorox** [reply]
@MatheusFarias8 done! star my repo pls https://github.com/malmorox/Portfolio

**Comment by akchurinslava** [comment]


**Reply by sanjisunster-cmd** [reply]
@akchurinslava [done](https://github.com/akchurinslava/car_app_mvc/stargazers)

you can star mine too [here](https://github.com/drknzz/GitHub-Achievements/discussions/274#discussion-6069600)

**Reply by sanjisunster-cmd** [reply]
> @sanjisunster-cmd heey :) no problem but I did not catch which repos?

@akchurinslava this [repo](https://github.com/rezzcode/zoea-opensource)

**Comment by spidybroz** [comment]
## ⭐ Star & 🍴 Fork the Repo <3

If you like this project, please **star** ⭐ the repo — I’ll star yours back!  
Feel free to **fork** 🍴 it, contribute, or share it with others 🚀

### 🔗 Links
- **GitHub:** https://github.com/maiz-an/DevTunnel-CLI  
- **Website:** https://devtunnel-cli.vercel.app

**Reply by spidybroz** [reply]
> @spidybroz Done, please star these: https://github.com/akchurinslava/car_app_mvc https://github.com/akchurinslava/django_rates https://github.com/akchurinslava/react_express https://github.com/akchurinslava/xamarin_to_do_list

done

**Reply by the1Riddle** [reply]
Hello, @maiz-an, you have a fantastic project! I love the concept, especially the fact that it works for both Node and PHP servers.

Do you have a short video demonstrating how it works? Sometimes we developers don't have time to install the package right away to experiment when it's first shared. At least that's how I felt when I reached the point of needing to copy the package command to test and run it on the website.

Also, many of us are already subscribed to this discussion, so you don’t need to repeat the same comment on everyone's commented posts @spidybroz. Starting with one comment like this and tagging everyone whose repo you’ve starred is a cleaner and more efficient way to notify everyone.

That's all I have... and btw, I've also started your projects since I found it to be amazing, great job pall.

If you find time, you can [star mine](https://github.com/rezzcode/zoea-opensource) too, thanks.

**Reply by spidybroz** [reply]
@the1Riddle sorry for it BTW i started, thanks

**Reply by maiz-an** [reply]
@the1Riddle 
 Hey! Thank you so much for your concern — I really appreciate it.

We do have a plan to create a video, but we’re currently updating the project to work with OS-native commands (winget, brew, sudo) instead of relying on npm. Our goal is to support users who don’t have Node.js installed, such as those who primarily work with Laravel or only use HTML and CSS.

Also, this tool won’t touch your source code at all — it only looks for the running port.

Once that update is complete, we plan to release a video that covers everything in detail.

**Reply by the1Riddle** [reply]
> @the1Riddle Hey! Thank you so much for your concern — I really appreciate it.
> 
> ....
> 
> Once that update is complete, we plan to release a video that covers everything in detail.

awesome 💪

**Comment by Mattral** [comment]
Let's exchange stars:
https://github.com/Mattral/Research-Search

**Reply by the1Riddle** [reply]
> @Mattral 

hello bruv

**Reply by christianalberto** [reply]
https://github.com/christianalberto/github-profile-achievements?utm_source=gemini

**Reply by the1Riddle** [reply]
> https://github.com/christianalberto/github-profile-achievements?utm_source=gemini

This was an eye-opener; I never knew that other badges existed apart from the ones listed in this repository @christianalberto

**Reply by christianalberto** [reply]
Can you give a star?

**Reply by the1Riddle** [reply]
> Can you give a star?

why not

**Reply by the1Riddle** [reply]
> @christianalberto 

Your badge is probably waiting 😄 

Mind checking this [project](https://github.com/rezzcode/Epubespiar)? It's not fully open source yet, but you can try the product itself

**Reply by christianalberto** [reply]
Cool
